package.json

```diff
@@ -31,7 +29,11 @@
   "devDependencies": {
     "@types/big.js": "^4.0.5",
     "@types/jest": "^24.0.25",
-    "tsdx": "^0.14.1"
+    "tslib": "^2.5.1",
```

Comment:
Do we need this library? I haven't used it before in any Typescript project

Reply:
This is for --importHelpers